wsdl/ver10/recording.wsdl

```diff
@@ -683,6 +690,36 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:attribute>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
+			<xs:element name="OverrideSegmentDurationRequest">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first ONVIF then Vendor -->
+					</xs:sequence>
+					<xs:attribute name="TargetSegmentDuration" type="xs:duration" minOccurs="1">
```

Comment:
Request root elements must not have xs:any as these result in ugly stubs. For other cases always position the xs:any at the last position as it otherwise eats all subsequent items.

Reply:
Fixed